### DISCUSSION TITLE
Suggested feature request template

### DISCUSSION THREAD

**Discussion by quickfold** [freeplane/freeplane#2152] [General]
I have noticed that many feature requests are missing information that @dpolivaev wants and asks for in response to the initial request. I put a few of his responses into ChatGPT and asked it to provide a template people could use to give better initial feature requests. I offer the result below (it could be automatically added to posts in the "Ideas" category). The examples aren't great, but the descriptions seem good to me. Feel free to use it, revise it, or ignore it as you see fit:

**Freeplane Feature Request**

1. **Feature Overview**
  Provide a brief description of the feature you are requesting. Explain the problem it solves and the outcome you hope for.
  _Example: "Add a batch rename function for tags. This would help maintain consistency and save time when reorganizing large projects."_

1. **Current Workflow Challenges:**
   Describe how you currently handle the task and what issues you face.
   _Example: I manually search for each tag and rename nodes one by one, which is tedious and error-prone._

1. **Proposed Functionality**
  Explain how the feature should work. Include detailed steps or examples of its expected behavior.
  _Example: "A feature that allows selecting a tag, entering a new name, and applying the change across the entire map."_

1. **Use Case and Expected Benefits**
  Explain how the feature would fit into your real-life workflow and what practical benefits it would provide.
  _Example: "This feature would streamline my process, save time, and ensure consistent tag usage across complex projects."_

1. **Example map and Supporting Materials**
  Attach a sample map where this feature would be useful, as will as relevant screenshots, mockups, or files that illustrate your request.
  _Example: "Attached is a screenshot of my current map and a mockup of how the feature could look."_

**Comment by dpolivaev** [comment]
Thanks, I updated https://github.com/freeplane/freeplane/blob/1.12.x/.github/ISSUE_TEMPLATE/feature_request.md based on your idea.
